src/zui/l10n/messageIds.ts

```diff
@@ -124,6 +124,7 @@ export default makeMessages('zui', {
     journeys: m('Journeys'),
     people: m('People'),
     projects: m('Projects & Activities'),
+    search: m('Search'),
```

Comment:
I don't see an update to the yml so i think you should rum `yarn mak-yaml` and generate the eng yaml file!

Reply:
I don't think that's necessary until it's time to publish the strings for translations, at which point I can publish them all in one go.